### ISSUE TITLE
Ask about moleculeACE

### ISSUE THREAD

**Issue by lingcon01** [GHUSER-haoyu/qcGEM#2]
Hello, I have a question regarding the results on the MoleculeACE dataset. I noticed that some of the baseline performances reported in your evaluation differ from those provided in the KPGT paper. For example, although both studies evaluate GOVER, the RMSE reported in your results appears to be noticeably higher than that reported in KPGT.

I was wondering whether this difference could be attributed to variations in data preprocessing, train–test splits, hyperparameter settings, or evaluation protocols.

Thank you very much for your time and clarification.

<img width="1654" height="1100" alt="Image" src="https://github.com/user-attachments/assets/5749fe28-aaa4-4855-860a-1678445d5402" />

<img width="827" height="695" alt="Image" src="https://github.com/user-attachments/assets/00177933-4046-4469-a96f-604d064c2d05" />

**Comment by GHUSER-haoyu** [comment]
I sincerely apologize for the delayed response. Thank you for raising this insightful question!

Regarding the performance discrepancies you observed (which may not be limited to the MoleculeACE benchmark), here is our detailed response:

To answer your main concerns directly:

1. Upon receiving your feedback, we thoroughly double-checked our code and data (including the code logic and the latent space distribution of the embeddings). Given the massive workload of this project, we did identify a few evaluation issues caused by oversight. We have recently updated the data and will release the fixed version in the future. However, please note that these newly identified issues do not account for the specific data discrepancy you mentioned.

2. As you correctly pointed out, the specific numerical differences between our results and KPGT's are highly likely due to variations in certain experimental settings. In fact, even if we perfectly aligned our settings with KPGT, discrepancies with other related works might still occur. What we can guarantee is that within our own paper, the comparisons across all baseline methods were conducted under strictly fair and identical settings.

3. Achieving absolute numerical consistency across different works is notoriously difficult in this field; typically, only overall trends or rankings remain consistent. Countless hyperparameters can influence the final outcome. Sometimes, even a slight change in the learning rate can lead to significant performance variations. This perspective has been widely echoed by numerous "systematic evaluation" and "rethinking" papers recently published in our community.

-------------------------------------------------

Nevertheless, to provide you with more context, we have brainstormed a few potential factors specific to our implementation that might explain these numerical differences:

1. Feature Extraction Strategy: We used frozen embeddings rather than full fine-tuning. Previous literature has explicitly shown that pre-trained models are extremely sensitive to the fine-tuning learning rate. We were not confident in finding the optimal fine-tuning learning rate for every single method via exhaustive grid search.

2. Downstream Evaluator: The architecture of the downstream model we used to process the embeddings and make predictions might differ from the one used in KPGT.

3. Evaluation Protocol: We evaluated a total of 30 method combinations across 83 datasets (with 11 newly added recently). Due to computational limits, we couldn't iterate through all possible evaluation protocols. We followed standard practices from prior works: simply taking the best validation performance within a fixed 100 epochs.

4. Selection of GROVER Features: For the GROVER-Large and Base models, we extracted their most comprehensive feature representations (i.e., both modes, incorporating both atom- and bond-level messages). We are unsure which specific mode KPGT utilized (they could have used the atom or bond mode). Given GROVER's extremely high dimensionality and the massive scale of our evaluations, saving features for all modes would cause severe storage issues. Therefore, we solely used the strongest mode as its representative.

We hope this information provides some clarity and is helpful for your research. If you have any further questions or would like to discuss this more, please feel free to follow up!
